### PR TITLE
removing FastCGI support

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
This PR removes FastCGI support (deprecated in https://github.com/openmicroscopy/openmicroscopy/pull/4056) in favor of WSGI as FastCGI support is deprecated since version 1.7 and will be removed in Django 1.9.

### CODE REVIEW

**Comment by joshmoore** [comment]
NB: Testing today can progress, but for tomorrow this will conflict.

File: components/tools/OmeroWeb/requirements.txt [hunk 1]
```diff
@@ -5,4 +5,5 @@
 #
 #     pip install -r requirements.txt
 #
-Django==1.8.4
\ No newline at end of file
+Django==1.8.4
+gunicorn>=19.3.0
```
Comment (sbesson) [inline components/tools/OmeroWeb/requirements.txt]: Arguably we should add the same line to `requirements_python26.txt`?

Comment (aleksandra-tarkowska) [inline components/tools/OmeroWeb/requirements.txt]: yes, done

**Comment by sbesson** [comment]
The content of this PR looks sensible at a glance (in agreement with the decision of dropping FastCGI). Note it depends on the merging of https://github.com/openmicroscopy/openmicroscopy/pull/4245 which moves the Web templates to a subfolder. Some related work will have to happen on the auto-generated documentation script /cc @hflynn

**Comment by hflynn** [comment]
There'll need to be a docs PR too

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-DEV-merge-push#114](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-push/114/). See the [console output](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-push/114/consoleText) for more details.
Possible conflicts:
  - Conflict detection failed
    - etc/templates/web/apache-fcgi.conf.template
    - etc/templates/web/apache-wsgi.conf.template
    - etc/templates/web/nginx-wsgi-development.conf.template
    - etc/templates/web/nginx-wsgi.conf.template

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-DEV-merge-push#115](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-push/115/). See the [console output](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-push/115/consoleText) for more details.
Possible conflicts:
  - Conflict detection failed
    - etc/templates/web/apache-fcgi.conf.template
    - etc/templates/web/apache-wsgi.conf.template
    - etc/templates/web/nginx-wsgi-development.conf.template
    - etc/templates/web/nginx-wsgi.conf.template

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-DEV-merge-push#116](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-push/116/). See the [console output](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-push/116/consoleText) for more details.
Possible conflicts:
  - Conflict detection failed
    - etc/templates/web/apache-fcgi.conf.template
    - etc/templates/web/apache-wsgi.conf.template
    - etc/templates/web/nginx-wsgi-development.conf.template
    - etc/templates/web/nginx-wsgi.conf.template

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-DEV-merge-push#117](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-push/117/). See the [console output](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-push/117/consoleText) for more details.
Possible conflicts:
  - Conflict detection failed
    - etc/templates/web/apache-fcgi.conf.template
    - etc/templates/web/apache-wsgi.conf.template
    - etc/templates/web/nginx-wsgi-development.conf.template
    - etc/templates/web/nginx-wsgi.conf.template

**Comment by aleksandra-tarkowska** [comment]
Closing in favor of #4283

**Pull request closed without merging**